yesod-core/src/Yesod/Core/Types.hs

```diff
@@ -35,9 +35,11 @@ import           Data.Serialize                     (Serialize (..),
 import           Data.String                        (IsString (fromString))
 import           Data.Text                          (Text)
 import qualified Data.Text                          as T
+import           Data.Text.Encoding
 import qualified Data.Text.Lazy.Builder             as TBuilder
 import           Data.Time                          (UTCTime)
 import           GHC.Generics                       (Generic)
+import qualified GHC.Int                            as I
```

Comment:
`GHC.Int` is kind of an internal module - generally preferable to import from `Data.Int` which is less GHC internals-y